realsense_camera/launch/zr300_nodelet_default.launch

```diff
@@ -11,10 +11,14 @@
   <arg name="mode"              default="manual" />
   <arg name="color_fps"         default="30" />
   <arg name="fisheye_fps"       default="30" />
+  <arg name="fisheye_strobe"    default="1" />
+  <arg name="fisheye_external_trigger"    default="1" />
 
   <param name="$(arg camera)/driver/mode"              type="str"  value="$(arg mode)" />
   <param name="$(arg camera)/driver/color_fps"         type="int"  value="$(arg color_fps)" />
   <param name="$(arg camera)/driver/fisheye_fps"       type="int"  value="$(arg fisheye_fps)" />
+  <param name="$(arg camera)/driver/fisheye_strobe"    type="int"  value="$(arg fisheye_strobe)" />
```

Comment:
This is one way of doing it. But to keep it consistent with how rest of the static parameters are set in the nodelet, can we have the defaults set in the function **ZR300Nodelet::getParameters()** instead of in the launch file?